### DISCUSSION TITLE
Shift (or control) + Click to select a list of objects

### DISCUSSION THREAD

**Discussion by 4l3m4oO** [friction2d/friction#44] [Ideas]
![image](https://github.com/friction2d/friction/assets/357488/46d2a4f7-2138-47e3-b6dc-069420e55c64)

Some times i want to select one list of objects, and i have to select them one by one. 

Maybe if there's some way to shit+click or ctrl+shift+click to select many objects in a row, it will improve the workflow of the projects.

Actually, it's possible only select all, deselect all, clear selection, invert selection or select one by one.

**Comment by AlmostMikhail** [comment]
We still need to implement this feature

**Comment by amberceo** [comment]
bump
this still hasnt been implemented, and i'm in support of it

**Comment by samytichadou** [comment]
this has been discussed as part of a rewrite of the shorcut manager
[here is the corresponding issue](https://github.com/friction2d/friction/issues/745)
:)
